base/abstractarray.jl

```diff
@@ -905,6 +905,8 @@ If `dst` and `src` are of the same type, `dst == src` should hold after
 the call. If `dst` and `src` are multidimensional arrays, they must have
 equal [`axes`](@ref).
 
+Behavior can be unexpected when any mutated argument shares memory with any other argument.
+
```

Comment:
The PR introduces heaps of repetition in the docstring. What I would suggest is defining a constant somewhere, like:

```
const _DOCS_ALIASING_WARNING = """
Behavior can be unexpected when any mutated argument shares memory with any other argument.
"""
```

And the just interpolating it into the docstring with `$(_DOCS_ALIASING_WARNING)` into each of the docstrings. Will make it easier to update or clarify the docstring in the future.

Reply:
done, thanks for the suggestion! not sure I put it in the right place though since we might need it outside of Base